introduction.md

```diff
@@ -4,6 +4,20 @@ this is roxana
 :wq!
 
 :wq
+:i
+*Git
+*JavaScript
+
+:wq
+
+:wq
+
+
+
+
+
+
+
 
 
 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?